### PR TITLE
feat: Added charts overrides

### PR DESCRIPTION
**Pull request by carlosgit2016**

### CODE REVIEW

File: charts-overrides/argocd-apps/values.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  finalizers:
+  - resources-finalizer.argocd.argoproj.io
+  project: default
+  sources:
+    - repoURL: https://kubernetes.github.io/ingress-nginx
+      targetRevision: 4.8.3
+      chart: ingress-nginx
+      helm:
+        valueFiles:
+        - $values/charts-overrides/nginx-controller/values.yaml
```
Comment (benhur-araujo) [inline charts-overrides/argocd-apps/values.yaml:16]: how this works?

File: charts-overrides/argocd-apps/values.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    - repoURL: https://github.com/conquer-project/helm-charts.git
+      targetRevision: feat/charts-overrides
```
Comment (benhur-araujo) [inline charts-overrides/argocd-apps/values.yaml]: main

File: charts-overrides/nginx-controller/values.yaml
Comment (benhur-araujo) [inline charts-overrides/nginx-controller/values.yaml:1]: is this just a test file?

Comment (CFlorr) [reply charts-overrides/nginx-controller/values.yaml:1]: I'll remove afterwards

**Review by benhur-araujo — APPROVED** [review]

**Pull request merged**